container/lxd/initialisation.go

```diff
@@ -122,19 +124,36 @@ func configureLXDProxies(setter configSetter, proxies proxy.Settings) error {
 	return nil
 }
 
-var execCommand = exec.Command
+// df returns the number of free bytes on the file system at the given path
+var df = func(path string) (uint64, error) {
```

Comment:
Why not make a simple function and make it consistent with the rest of the code, instead of using this lambda?